Environment_Engine/Query/Absorptance.cs

```diff
@@ -41,33 +41,43 @@ public static partial class Query
         /**** Public Methods                            ****/
         /***************************************************/
 
-        [Description("Returns the total Absorptance of a construction")]
+        [Description("Returns the absorptance of a construction")]
         [Input("construction", "A Construction object")]
-        [Output("absorptance", "The total absorptance of the construction")]
-        public static Absorptance Absorptance(this Construction construction)
+        [Output("absorptance", "The total absorptance of all materials on the construction")]
+        public static double Absorptance(this Construction construction)
```

Comment:
That one will need further work, this is a place holder to allow the XML Toolkit to compile and continue working as the XML Toolkit currently isn't doing constructions for IES while we wait IES responses to some queries. So this will have to stay as is for now.